public/app/features/provisioning/ConfigForm.tsx

```diff
@@ -0,0 +1,163 @@
+
+            <p>Ensure that your token has the following permissions:</p>
+
+            <b>For all repositories:</b>
+            <pre>public_repo, repo:status, repo_deployment, read:packages, read:user, user:email</pre>
+
+            <b>For GitHub projects:</b>
+            <pre>read:org, read:project</pre>
+
+            <b>An extra setting is required for private repositories:</b>
+            <pre>repo (Full control of private repositories)</pre>
+          </ControlledCollapse>
+          <Field label={'GitHub token'} required error={errors?.token?.message} invalid={!!errors.token}>
+            <Controller
+              name={'token'}
+              control={control}
+              rules={{ required: 'This field is required.' }}
+              render={({ field: { ref, ...field } }) => {
+                return (
+                  <SecretInput
+                    {...field}
+                    id={'token'}
+                    placeholder={'ghp_yourTokenHere1234567890abcdEFGHijklMNOP'}
+                    isConfigured={tokenConfigured}
+                    onReset={() => {
+                      setValue('token', '');
+                      setTokenConfigured(false);
+                    }}
+                  />
+                );
+              }}
+            />
+          </Field>
+          <Field label={'Repository owner'} error={errors?.owner?.message} invalid={!!errors?.owner}>
+            <Input {...register('owner', { required: 'This field is required.' })} placeholder={'test'} />
+          </Field>
+          <Field label={'Repository name'} error={errors?.repository?.message} invalid={!!errors?.repository}>
+            <Input {...register('repository', { required: 'This field is required.' })} placeholder={'example'} />
+          </Field>
+          <Field label={'Commit directly to main'}>
+            <Switch {...register('branchWorkflow')} id={'branchWorkflow'} />
+          </Field>
+          <Field label={'Show dashboard previews'}>
+            <Switch {...register('generateDashboardPreviews')} id={'generateDashboardPreviews'} />
+          </Field>
+        </>
+      )}
+
+      {watchType === 'local' && (
+        <Field label={'Local path'} error={errors?.path?.message} invalid={!!errors?.path}>
+          <Input {...register('path', { required: 'This field is required.' })} placeholder={'/path/to/repo'} />
+        </Field>
+      )}
+
+      {watchType === 's3' && (
+        <>
+          <Field label={'S3 bucket'} error={errors?.bucket?.message} invalid={!!errors?.bucket}>
+            <Input {...register('bucket', { required: 'This field is required.' })} placeholder={'bucket-name'} />
+          </Field>
+          <Field label={'S3 region'} error={errors?.region?.message} invalid={!!errors?.region}>
+            <Input {...register('region', { required: 'This field is required.' })} placeholder={'us-west-2'} />
+          </Field>
+        </>
+      )}
+      <Field label={'Resource folder'}>
```

Comment:
```suggestion
      <Field label={'Target folder'}>
```
Resources will be loaded into this folder (if appropriate) -- eg some "things" do not belong to folders (playlists, etc)